SimPEG/regularization/base.py

```diff
@@ -422,41 +426,40 @@ def f_m_deriv(self, m) -> csr_matrix:
 
     @utils.timeIt
     def deriv(self, m) -> np.ndarray:
-        r"""Gradient of the regularization function evaluated for the model provided.
+        r"""Jacobian of the regularization function evaluated for the model provided.
```

Comment:
Leave it as Gradient, this function does not return a Jacobian (a matrix) it returns a vector

Reply:
agreed